libnet/src/libnet_link_pf.c

```diff
@@ -49,22 +49,25 @@ libnet_open_link_interface(int8_t *device, int8_t *ebuf)
     l = (struct libnet_link_int *)malloc(sizeof(*l));
     if (l == NULL)
     {
-        sprintf(ebuf, "libnet_open_link_int: %s", strerror(errno));
+                 snprintf(ebuf, LIBNET_ERRBUF_SIZE,
+"libnet_open_link_int: %s", strerror(errno));
         return (0);
     }
     memset(l, 0, sizeof(*l));
     l->fd = pfopen(device, O_RDWR);
     if (l->fd < 0)
     {
-        sprintf(ebuf, "pf open: %s: %s\n\your system may not be properly configured; see \"man packetfilter(4)\"",
+                 snprintf(ebuf, LIBNET_ERRBUF_SIZE,
+"pf open: %s: %s\n\your system may not be properly configured; see \"man packetfilter(4)\"",
```

Comment:
indentation, and existing bug: `\y`, since you are touching the line, could you fix that, please?